.docforge/documentation/other-components.yaml

```diff
@@ -35,3 +35,14 @@ structure:
   - fileTree: https://github.com/gardener/dependency-watchdog/tree/master/docs
     excludeFiles:
     - "README.md"
+- dir: gardener-discovery-server
+  structure:
+  - file: _index.md
+    frontmatter:
+      title: Gardener Discovery Server
+      weight: 1
+      description: A server which provides public metadata about different Gardener resources
```

Comment:
```suggestion
      description: A server capable of serving public metadata about different Gardener resources like shoot OIDC discovery documents and Gardener Workload Identity discovery
```

Reply:
The suggested description is too long and copies the first sentence of the topic word for word, which is not recommended for short descriptions. I would even change it to something like: "A server providing public metadata for Gardener resources."